AK/IPv6Address.h

```diff
@@ -270,18 +270,8 @@ static_assert(sizeof(IPv6Address) == 16);
 
 template<>
 struct Traits<IPv6Address> : public GenericTraits<IPv6Address> {
-    static constexpr unsigned hash(IPv6Address const& address)
-    {
-        unsigned h = 0;
-        for (int group = 0; group < 8; group += 2) {
-            u32 two_groups = ((u32)address[group] << 16) | (u32)address[group + 1];
-            if (group == 0)
-                h = int_hash(two_groups);
-            else
-                h = pair_int_hash(h, two_groups);
-        }
-        return h;
-    }
+    // SipHash-4-8 is considered conservatively secure.
```

Comment:
"...for HashDoS prevention purposes."

Reply:
As a general-purpose hash function it has relatively high collision resistance in general, so I'm not sure that is accurate either. I will find a better wording.